src/page-app/ClientSideRoutes.tsx

```diff
@@ -0,0 +1,62 @@
+/**
+ * Important: This path is initially created in the gatsby-config.js file using the gatsby-plugin-create-client-paths.
+ * The pathname params are accessed because of the url fragment in that file.
+ */
+
+import React from "react";
+import {
+  BrowserRouter as Router,
+  Redirect,
+  Route,
+  RouteComponentProps,
+  Switch,
+} from "react-router-dom";
+
+import { Routes } from "../constants/Routes";
+import SiteHeader from "../site-header/SiteHeader";
+import DashboardContainer from "./DashboardContainer";
+import FacilityContainer from "./facility/FacilityContainer";
+import ScenarioContainer from "./scenario/ScenarioContainer";
+import Scenario from "./scenario/ScenarioContent";
+
+interface FacilityParamProps {
+  scenarioId: string;
+  facilityId: string;
+}
+
+const ClientSideRoutes = () => {
+  return (
+    <DashboardContainer>
+      <Router>
+        <SiteHeader isApp={true} />
+        <Switch>
+          <Route
+            path={Routes.Facility.url}
+            render={({ match }: RouteComponentProps<FacilityParamProps>) => {
+              const { scenarioId } = match.params;
```

Comment:
`match.params` here should contain both `scenarioId` and `facilityId` (when I run this locally and log it, that's what I see) ... why can't we just pass them as props to the UI component we want to render instead of parsing the url again in the FacilityContainer? (this same comment may apply to all these routes; I'm not sure, I haven't had time to test them extensively)